src/display/editor/freetext.js

```diff
@@ -330,12 +365,18 @@ class FreeTextEditor extends AnnotationEditor {
 
     super.render();
     this.editorDiv = document.createElement("div");
-    this.editorDiv.tabIndex = 0;
     this.editorDiv.className = "internal";
 
-    FreeTextEditor._l10nPromise.then(msg =>
-      this.editorDiv.setAttribute("default-content", msg)
-    );
+    this.editorDiv.id = `${this.id}-editor`;
```

Comment:
This is another case where I'm not sure that it's *generally* a good idea to add element `id`s to the DOM, since that makes them linkable.
Similar to other recent patches, can we use a *custom* `data-element-id` attribute instead?

(Also, if there's other pre-existing cases of `id`s being used in the Editor they probably need to be updated as well.)

Reply:
Unfortunately `aria-owns` value must be an `id` or a list of ids:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-owns#values
All the ideas are prefixed by:
https://github.com/mozilla/pdf.js/blob/e08b07969191668915fb3111b2487dd07502b222/src/shared/util.js#L55
We can make it a bit more complex and dynamic to avoid any conflicts.